### DISCUSSION TITLE
Preservation Project

### DISCUSSION THREAD

**Discussion by shaykbake** [shaykbake/Tribes1Preservation#2] [General]
If you have something you want to offer here, let me know. We can work on getting it added to the repo.

Even with a few large-scale online sources for Tribes files still being around as of late, it seems unlikely we'll have this stuff forever. I figured preserving it on Github was the way to go.

Just keep in mind that Github has a 25MB file size limit, so nothing over that is going to work unless we can split it into parts for recombining after download. That's likely how I'll work out uploading installations and the like.

Thanks for helping keep this stuff alive. Outside of a few slightly active T1 servers, there isn't much these days, and it's a shame given how influential this game was to a lot of what is coming out even still today, and how fun of a game it can still be when a decent crowd are able to get onto a server together!

**Comment by matrix8967** [comment]
Heya!

I still have *all* the mods, skins, maps, scripts, etc. that I downloaded for Tribes as a kid.

I also have a lot of the DevTools that have made the rounds in various releases.

It's all pretty messy - because I was a kid and had no idea what I was doing...

But I think I've got a lot of the good stuff packaged up if you wanna take a look.

If you want I'll just make a PR and we can chop it up that way - just let me know.

Here's a directory tree of the [Dev-Tools directory](https://keybase.pub/matrix8967/Tribes/Starsiege_Tribes_Tools_Directory_Tree.md), and one for the [Mods directory.](https://keybase.pub/matrix8967/Tribes/Starsiege_Tribes_Mods_Directory_Tree.md)

I'm actually writing a script to go pull all the `urls` from the `webarchive`/`wayback machine`, then try to download anything that is lucky enough to have gotten archived - but that'll probably be later this week.

**Reply by shaykbake** [reply]
Cool deal. I'd definitely love to see what you have, and am glad to see others have held on to what they managed to capture back when these files were available. This sounds sort of like my situation: I have hordes of files that were accumulated over a span of several active Tribes years without any real organization to them. At least the mods are somewhat filed in directories (still a bit messy), but a lot of the other stuff (scripts, tools, maps, etc.) are going to be a bit time consuming.

In addition to that, the files that I have were moved across several drives over the past 20+ year span, so I'm just hoping that I'm not going to find too many of them corrupted when attempting to extract archives, etc.

**Reply by matrix8967** [reply]
Finally got around to this -- just submitted a PR that has all my archives. Details are in the repo.

Feel free to take what you need or purge anything you don't want or is redundant.

**Comment by griffincloud** [comment]
Thanks for doing this, great project. Most of my old links are dead and lots of files/info have been lost over the years.
